framework/include/material_image_helpers.hpp

```diff
@@ -88,80 +103,226 @@ namespace gvk
 		return result;
 	}
 
-	// create cubemap from image data
+	/** Create cube map from image_data, with optional caching
+	* Loads image data from an image_data object or the serializer cache.
+	* @param aImageData		a valid instance of image_data containing cube map image data.
+	* @param aMemoryUsage	the intended memory usage of the returned image resource.
+	* @param aImageUsage	the intended image usage of the returned image resource.
+	* @param aSyncHandler	the sync handler to use creating the image resource.
+	* @param aSerializer	a serializer to use for caching data loaded from disk. 
+	*/
 	avk::image create_cubemap_from_image_data_cached(image_data& aImageData, avk::memory_usage aMemoryUsage = avk::memory_usage::device,
 		avk::image_usage aImageUsage = avk::image_usage::general_cube_map_texture, avk::sync aSyncHandler = avk::sync::wait_idle(), std::optional<std::reference_wrapper<gvk::serializer>> aSerializer = {});
 
+	/** Create cube map from image_data, with caching
+	* Loads image data from an image_data object or the serializer cache.
+	* @param aSerializer	a serializer to use for caching data loaded from disk. 
+	* @param aImageData		a valid instance of image_data containing cube map image data.
+	* @param aMemoryUsage	the intended memory usage of the returned image resource.
+	* @param aImageUsage	the intended image usage of the returned image resource.
+	* @param aSyncHandler	the sync handler to use creating the image resource.
+	*/
 	static avk::image create_cubemap_from_image_data_cached(gvk::serializer& aSerializer, image_data& aImageData, avk::memory_usage aMemoryUsage = avk::memory_usage::device, 
 		avk::image_usage aImageUsage = avk::image_usage::general_cube_map_texture, avk::sync aSyncHandler = avk::sync::wait_idle())
 	{
 		return create_cubemap_from_image_data_cached(aImageData, aMemoryUsage, aImageUsage, std::move(aSyncHandler), aSerializer);
 	}
 
+	/** Create cube map from image_data
+	* Loads image data from an image_data object.
+	* @param aImageData		a valid instance of image_data containing cube map image data.
+	* @param aMemoryUsage	the intended memory usage of the returned image resource.
+	* @param aImageUsage	the intended image usage of the returned image resource.
+	* @param aSyncHandler	the sync handler to use creating the image resource.
+	*/
 	static avk::image create_cubemap_from_image_data(image_data& aImageData, avk::memory_usage aMemoryUsage = avk::memory_usage::device,
 		avk::image_usage aImageUsage = avk::image_usage::general_cube_map_texture, avk::sync aSyncHandler = avk::sync::wait_idle())
 	{
 		return create_cubemap_from_image_data_cached(aImageData, aMemoryUsage, aImageUsage, std::move(aSyncHandler));
 	}
 
-	// create cubemap from a single file
+	/* create cube map from a single file, with optional caching
```

Comment:
```suggestion
	/** Create cube map from a single file, with optional caching
```

Reply:
Many comments are now missing the double asterisk. But it is important to have them since they mark the beginning of a special comment (for Doxygen and suchlike).